src/actions/app.cr

```diff
@@ -106,12 +106,22 @@ module DppmRestApi::Actions::App
   # return the base application package
   relative_get "/:app_name/app" do |context|
     raise Unauthorized.new context unless Actions.has_access? context, Access::Read
-    # TODO: return the base application package
+    app_name = context.params.url["app_name"]
+    build_json context.response do |builder|
+      builder.field(app_name) do
+        prefix.new_app(app_name).to_json builder
```

Comment:
This doesn't make any sense, and the whole route should be deleted, right? Because all config data lies under `pkg`, right?

Reply:
That's right for the `pkg` information, but how do we know on what package the application is based?

Reply:
By the way the endpoint name is confusing, because on the filesystem `myapp/app` is a symlink to `pkg/app`.
I suggest to name it `pkg`, in reference of the `pkg` endpoint.


The actual `pkg` endpoints can be renamed to `pkg_info`.
I will make a breaking change in this sense to rename the `info` field in `pkg.con`, to avoid having `pkg_info/info` in the API.
Or do you think `pkg_file` will be fine in the API?